index.html

```diff
@@ -2096,7 +2096,12 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
           "rfc2119" title="MUST">MUST</em> include an entry in the array
           associated with the name <code>additionalResponses</code> that
           includes a value for the name <code>schema</code>.</span> 
-          </p></section>
+          </p><p>
+          A <a>Thing Description</a> of a WoT producer may has multiple forms entries with, e.g., different 
+          protocol and/or content types declarations that a <a>Consumer</a> would also support. In that case
+          it is recommended that the <a>Consumer</a> chooses the first form entry that works for them.
```

Comment:
I think this should be formulated in more generic terms, something like "In that case the <a>Consumer</a> may choose any form entry that works for them".

Reply:
Should we also suggest a retaining policy? For example "When one form is chosen, it is expected that the consumer will continue to use it as long as possible for every new interaction with the Producer" ? 🤔  

Also _works for them_ feels a little bit vague but I understand that we might not need to be specific about implementation details.  What happens if the form chosen is unreachable? The consumer should give up or try again with another form?

Reply:
@zolkis 
>I think this should be formulated in more generic terms, something like "In that case the Consumer may choose any form entry that works for them".

I thought the idea was to provide a proposal, which one the Consumer should select if there are multiple possible.

@relu91 
>"When one form is chosen, it is expected that the consumer will continue to use it as long as possible for every new interaction with the Producer"

+1

Reply:
Yes, but "the first form entry that works for them" is equivalent to "any form entry".
Why is it a problem if the implementation chooses the form? (the one best matching its own context).
If we have generic guidelines for specific cases, we should be explicit about them, like when choosing between protocols, which would need defining a priority list for protocols (just an example).